box-ui/src/components/JitsiComponent.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import './JitsiMeet.css';
+
+const JitsiMeetComponent = () => {
```

Comment:
Put the style const in a css file. In each div, we declare a className that will be styled in the css file.